internal/databases/mysql/stream_fetch_handler.go

```diff
@@ -45,21 +45,66 @@ func FetchLogs(folder storage.Folder, backup *internal.Backup) error {
 			backupUploadTime = binlog.GetLastModified()
 		}
 	}
-	dstFolder, fetchedBinlogs, err := internal.FetchLogs(folder, backupUploadTime, BinlogFetchSettings{})
```

Comment:
If I got it right. You want to filter logs. So it would be better to update FetchLogs and pass filter func as argument.